server/plugins/TriggerUGens.cpp

```diff
@@ -775,8 +774,7 @@ void Poll_Ctor(Poll* unit) {
 
     unit->m_id_string[idSize] = '\0';
     unit->m_mayprint = unit->mWorld->mVerbosity >= -1;
-
```

Comment:
I have no clue where this suddenly comes from. It is not visible in my own branch …

Reply:
This seems to come from commit 3a7af1564eed65ec8ffbc2c3a271710ce73e2ecd